scripts/mixedos_sonobuoy.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
```

Comment:
i am getting this error when i try to run whitout delete the sonobuoy 
```
•failed to execute install action sonobuoy: exit status 1
Output: Installing sonobuoy version 0.56.16 for mixedos validation
fatal: destination path 'my-sonobuoy-plugins' already exists and is not an empty directory.
x LICENSE
x sonobuoy
```

shoulndt we need an automatic deletion  ?   or is just on my end ?

Reply:
ok let me check that, I did not run it locally so did not have that issue... likely need to add a check for the folder

Reply:
Updated the script, please check and let me know if that worked

Reply:
checking

Reply:
Updated again, added sudo now...

Reply:
as discussed, will address this later